ogusa/parameters.py

```diff
@@ -220,24 +216,17 @@ def compute_default_params(self):
         # Calculations for business income taxes
         # at some point, we will want to make Cost of Capital Calculator
         # a dependency to compute tau_b
-        c_corp_share_of_assets = 0.55
         # this adjustment factor has as the numerator CIT receipts/GDP
-        # from US data and as the demoninator CIT receipts/GDP from the
+        # and as the demoninator CIT receipts/GDP from the
```

Comment:
@jdebacker . Here is another typo in `parameters.py`. This one is a good one: "demoninator".

Reply:
Ah - my spell check in VS code not highlighting issues in comments or docstrings!